tutorials/how_to_use_polsih_static_embeddings.ipynb

```diff
@@ -0,0 +1,795 @@
+      "        0., 0., 0., 0.])\n",
+      "Token: 11 czarne\n",
+      "tensor([0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.,\n",
+      "        0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.,\n",
+      "        0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.,\n",
+      "        0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.,\n",
+      "        0., 0., 0., 0.])\n",
+      "Token: 12 jest\n",
+      "tensor([-0.2300, -0.0609,  0.2554, -0.1358, -0.1616, -0.1697, -0.0422,  0.1749,\n",
+      "        -0.3816, -0.1687, -0.1137, -0.4439,  0.5215,  0.3072,  0.1272,  0.4861,\n",
+      "        -0.3711, -0.2262,  0.5448, -0.6074, -0.1269,  0.0770, -0.1696,  0.0153,\n",
+      "        -0.2848, -0.1207,  0.2115, -0.0878, -0.0392, -0.5609,  0.2343, -0.0376,\n",
+      "        -0.5522, -0.3446, -0.0009,  0.0193,  0.0437, -0.5025,  0.0141,  0.0955,\n",
+      "         0.0156,  0.0687, -0.0134, -0.0052,  0.1033, -0.3966,  0.0394,  0.0415,\n",
+      "        -0.2180,  0.0280, -0.1671,  0.1898, -0.0262,  0.2860, -0.3344,  0.3450,\n",
+      "         0.4080,  0.3890, -0.0316,  0.0837,  0.2744,  0.0749, -0.5411, -0.1273,\n",
+      "        -0.6650, -0.5286,  0.2065,  0.0562, -0.1420,  0.1886,  0.1490, -0.1750,\n",
+      "         0.2962,  0.1380,  0.1489, -0.0349,  0.1351, -0.0132,  0.1280, -0.0559,\n",
+      "         0.4006, -0.2825,  0.1845, -0.1669,  0.2481,  0.0039, -0.3598,  0.3710,\n",
+      "        -0.1417, -0.0138, -0.0090,  0.1456,  0.1863, -0.0881, -0.1819,  0.3967,\n",
+      "        -0.0889,  0.2459, -0.0658,  0.1421])\n",
+      "Token: 13 czarne\n",
+      "tensor([0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.,\n",
+      "        0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.,\n",
+      "        0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.,\n",
+      "        0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.,\n",
+      "        0., 0., 0., 0.])\n",
+      "Token: 14 .\n",
+      "tensor([0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.,\n",
+      "        0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.,\n",
+      "        0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.,\n",
+      "        0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.,\n",
+      "        0., 0., 0., 0.])\n"
+     ]
+    }
+   ],
+   "source": [
+    "for token in sentence:\n",
+    "    print(token)\n",
+    "    print(token.embedding)"
```

Comment:
add `torch.set_printoptions(threshold=8)` at the beginning of notebook to make it shorter